### ISSUE TITLE
dataset split

### ISSUE THREAD

**Issue by LoveYuxiaoyu** [nnzhan/MTGNN#31]
Hello, I have a question. In paper's appendix, it said"Following [13], we split these two datasets into a training set (70%),
validation set (20%), and test set (10%) in chronological order." But actually, DCRNN and the code is: training set (70%),
validation set (10%), and test set (20%). I wonder if this is a typo。Thanks!

**Comment by nnzhan** [comment]
Sorry, this is a typo. Please follow the codes.
